Client.Linq/Internal/Expressions/MeasurementColumnName.cs

```diff
@@ -0,0 +1,22 @@
+using System.Reflection;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions
+{
+    internal class MeasurementColumnName : IExpressionPart
+    {
+        private readonly ColumnName _delegate;
+
+        internal MeasurementColumnName(MemberInfo member, IMemberNameResolver memberNameResolver)
+        {
+            _delegate = new ColumnName(member, memberNameResolver);
+        }
+
+        public void AppendFlux(StringBuilder builder)
+        {
+            builder.Append("r[\"");
+            _delegate.AppendFlux(builder);
+            builder.Append("\"]");
+        }
+    }
+}
```

Comment:
Please, add test for filtering by measurement in LINQ driver

https://github.com/influxdata/influxdb-client-csharp/blob/a671601c727072a4d07466d04748ff9b16b52888/Client.Linq.Test/InfluxDBQueryVisitorTest.cs

Reply:
Something like will be pretty fine:

```csharp
public void FilterByMeasurement()
{
    var query = from s in InfluxDBQueryable<SensorWithCustomMeasurement>.Queryable("my-bucket", "my-org", _queryApi)
        where s.Value > 10
        where s.Measurement == "my-measurement"
        select s;
    var visitor = BuildQueryVisitor(query);

    const string expected = "start_shifted = int(v: time(v: p2))\n\nfrom(bucket: p1) " +
                            "|> range(start: time(v: start_shifted)) " +
                            "|> filter(fn: (r) => (r[\"_measurement\"] == p4)) " +
                            "|> pivot(rowKey:[\"_time\"], columnKey: [\"_field\"], valueColumn: \"_value\") " +
                            "|> drop(columns: [\"_start\", \"_stop\", \"_measurement\"]) " +
                            "|> filter(fn: (r) => (r[\"data\"] > p3))";

    Assert.AreEqual(expected, visitor.BuildFluxQuery());
    
    var ast = visitor.BuildFluxAST();
    
    var measurementAssignment = ((OptionStatement) ast.Body[3]).Assignment as VariableAssignment;
    Assert.AreEqual("p4", measurementAssignment?.Id.Name);
    Assert.AreEqual("my-measurement", (measurementAssignment?.Init as StringLiteral)?.Value);
}
```

```csharp
public class SensorWithCustomMeasurement
{
    [Column(IsMeasurement = true)]
    public string Measurement { get; set; }

    [Column("sensor_id", IsTag = true)] 
    public string SensorId { get; set; }

    [Column("data")]
    public int Value { get; set; }
}
```

You have to update a [`QueryVisitor`](https://github.com/influxdata/influxdb-client-csharp/blob/25e9f7cf5404274c961442af95f6b876ce4c1b3b/Client.Linq/Internal/QueryVisitor.cs#L88) by:

![image](https://user-images.githubusercontent.com/455137/135399137-b103a881-6177-4e68-aa42-2fbb06e046b6.png)